spec/need_kconfig_format_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'spec_helper'
+require 'yaml'
+
+def check_config_name(name)
+  # Check if config name format is valid:
+  # 1. not start with CONFIG_
+  !name.to_s.start_with?('CONFIG_')
```

Comment:
thanks, prefer the logic to be
```
expect(name.to_s).to be_start_with('CONFIG_')
```
So it checks the expected pattern by expect method, and when it fails, the failure output could be more direct to show what the wrong name is.